### PR TITLE
Add mock-data and a little styling

### PR DESCRIPTION
**Pull request by Lint64**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/devrzu/frontrzu/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/app/feed/mock-feed.service.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export class FeedService {
+  constructor(
+    protected http: HttpClient,
+    public helperService: HelperService
+  ) {}
+
+  private keywordSource = new BehaviorSubject('default message');
+  keywordMessage = this.keywordSource.asObservable();
+
+  changeKeyword(message: string) {
```
Comment (eunchong) [inline src/app/feed/mock-feed.service.ts]: @Lint64 where is to use `changeKeyword` method?

File: src/app/feed/post/post.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+export class PostComponent implements OnInit {
+  @Input() post;
+  private comments = [];
+  private visibleAllComments = false;
+  constructor() {}
+  ngOnChanges() {
+    console.log(`post-component ngOnChanges`);
+  }
+
+  test() {
```
Comment (eunchong) [inline src/app/feed/post/post.component.ts]: why do you use this method? Let me know about test() plz.

File: src/app/feed/mock-feed.service.ts (hunk 1 above)
Comment (Lint64) [reply src/app/feed/mock-feed.service.ts]: This method was copied from [feed.service.ts](https://github.com/devrzu/frontrzu/commit/62ae844f3e63a883db1fac27d7cf93fcb754fec1#diff-9e41836db78fb07bbc2ed93fa35d650aR26) created by @eunchong

File: src/app/feed/post/post.component.ts (hunk 2 above)
Comment (Lint64) [reply src/app/feed/post/post.component.ts]: This is the test method written for the "read more" button.

**Comment by eunchong** [comment]
changeKeyword is my fault.. It's useless. 
I'll remove that.

File: src/app/feed/mock-feed.service.ts (hunk 1 above)
Comment (eunchong) [reply src/app/feed/mock-feed.service.ts]: 🐵

**Pull request merged**